### PR TITLE
Less broken config

### PR DESCRIPTION
**Pull request by ManiacMoustache**
A demo of config for robotmap values. It works in the 2020 sim with values pulled from config.yml inside the team3200 folder.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.PHONY: sim
```
Comment (tuffinmuffin) [inline Makefile]: this file was deprecated I think?

File: exit [hunk 2]
```diff
@@ -0,0 +1,51 @@
+[33mcommit 51eba9dd1f587b7b80c2e13cf53b7aef7b27dcec[m[33m ([m[1;36mHEAD -> [m[1;32mbasic-drivetrain[m[33m, [m[1;31mupstream/basic-drivetrain[m[33m)[m
```
Comment (tuffinmuffin) [inline exit]: This looks like a junk file. Be careful when adding files using wild cards. Each file added should be reviewed before commiting.

File: imgui.ini [hunk 3]
```diff
@@ -0,0 +1,145 @@
+[Window][Debug##Default]
```
Comment (tuffinmuffin) [inline imgui.ini]: FIle should not be commited, unless you mean to add a sim layout.

File: physics.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+#
```
Comment (tuffinmuffin) [inline physics.py]: depricated file. Should be deleted. May get deleted after merged with master.

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,16 @@
-pygame
```
Comment (tuffinmuffin) [inline requirements.txt]: Added back in old 2019 file. Please remove changes.

File: robot.py [hunk 6]
```diff
@@ -20,5 +59,4 @@
     except Exception as err:
         print("Failed to do extra setup. Error", err)
 
-    wpilib.run(Robot, physics_enabled=True)
-
+    wpilib.run(Robot, physics_enabled=True)
```
Comment (tuffinmuffin) [inline robot.py]: No new line at end of file

File: sim/config.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (tuffinmuffin) [inline sim/config.json]: depricated file. Should be deleted. May get deleted after merged with master.

File: team3200/motorHelper.py [hunk 8]
```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+
+import rev
+import ctre
+
+import logging
+log = logging.getLogger("console") #These logs were set up for testing, should not be persistent, please delete if you see these and I forgot
```
Comment (tuffinmuffin) [inline team3200/motorHelper.py]: Logger name should be realted to motorHelper. Logger level set to debug but I don't see any debug or high log statements?

File: Makefile (hunk 1 above)
Comment (ManiacMoustache) [reply Makefile]: Fixed

File: components/driveTrain.py [hunk 9]
```diff
@@ -1,5 +1,5 @@
 import wpilib.drive
-import motorHelper
+import motorHelper as motorHelper
```
Comment (tuffinmuffin) [inline components/driveTrain.py]: redudent rename. Are you trying to do from motorHelper import motorHelper?

File: __init__.py [hunk 10]
```diff
@@ -0,0 +1,11 @@
+# Import the base classes to use anywhere in team3200
```
Comment (tuffinmuffin) [inline __init__.py]: Shouldn't this file have been removed?

**Pull request merged**